converter/shieldhit/parser.py

```diff
@@ -174,9 +174,49 @@ def _parse_zones(self, json: dict) -> None:
                 figures_operators=self._parse_csg_operations(zone["unionOperations"]),
                 # lists are numbered from 0, but shieldhit materials are numbered from 1
                 material=self.geo_mat_config.materials.index(zone["materialData"]["id"])+1,
-            ) for idx, zone in enumerate(json["zonesManager"]["zones"])
+            ) for idx, zone in enumerate(json["zoneManager"]["zones"])
         ]
 
+        if "worldZone" in json["zoneManager"]:
+            self._parse_world_zone(json)
+
+    def _parse_world_zone(self, json: dict) -> None:
+        """Parse the world zone and add it to the zone list"""
+        # Add bounding figure to figures
+        world_figure = solid_figures.parse_figure(json["zoneManager"]["worldZone"])
+        self.geo_mat_config.figures.append(
+            world_figure
+        )
+
+        self.geo_mat_config.zones.append(
+            Zone(
+                id=len(self.geo_mat_config.zones)+1,
+                # slightly larger world zone - world zone
+                figures_operators=self._calculate_world_zone_operations(len(self.geo_mat_config.figures)),
+                # the last material is the black hole
+                material=1000
+            )
+        )
+
+        # Add the figure that will serve as a black hole wrapper around the bounding zone
+        world_figure = solid_figures.parse_figure(json["zoneManager"]["worldZone"])
+        world_figure.expand(1.)
```

Comment:
explain why `.expand(1.)`

Reply:
It expand

Reply:
Indeed...

Reply:
So add a comment explaining that you expand the zone by 1 cm margin or something like that